File: tasks/javascript/easy/sum.js
// JavaScript - Easy

// Function that returns the sum of two numbers
function sum(a, b) {
    return a + b;
}

// Example usage and testing
if (typeof module !== 'undefined' && module.exports) {
    module.exports = sum;
} else {
    // Browser environment - test the function
    console.log("Testing sum function:");
    console.log("sum(2, 3) =", sum(2, 3)); // Expected: 5
    console.log("sum(-1, 1) =", sum(-1, 1)); // Expected: 0
    console.log("sum(0, 0) =", sum(0, 0)); // Expected: 0
    console.log("sum(10.5, 5.5) =", sum(10.5, 5.5)); // Expected: 16
}

File: tasks/c++/easy/sum.cpp
// C++ - Easy 2

// TODO: Create a Basic C++ Program to Add Two Numbers
#include<iostream>
using namespace std;
int f(int a=0,int b=0){
	return a+b;
}
int main()
{
  int a,b;
  cin>>a>>b;
  cout<<f(a,b);
  return 0; 
}
